src/neuroglancer/trackable_mip_level_constraints.ts

```diff
@@ -0,0 +1,115 @@
+import {TrackableValue} from 'neuroglancer/trackable_value';
+import {RefCounted} from 'neuroglancer/util/disposable';
+import {verifyNonnegativeInt, verifyOptionalNonnegativeInt} from 'neuroglancer/util/json';
+import {NullarySignal} from 'neuroglancer/util/signal';
+
+export type TrackableMIPLevel = TrackableValue<number|undefined>;
+
+export class TrackableMIPLevelConstraints extends RefCounted {
+  minMIPLevel: TrackableMIPLevel;
+  maxMIPLevel: TrackableMIPLevel;
+  changed = new NullarySignal();
+  private numberLevels: number|undefined;
+  private dispatchOff = false;
```

Comment:
Think it's easier to read the code if you remove the negation in the variable name. E.g. you could call it `dispatchEnabled`?